### PR TITLE
Fix: bug Total Price should includes QC fee as well

### PR DESCRIPTION
**Pull request by luthfiazhari**
Fix: bug Total Price should includes QC fee as well 
[https://blocksphere2020.atlassian.net/browse/DBIO-405](url)

And fix wording Document to EMR NavigationDrawer when user access 'customer/request-test/find-lab'

### CODE REVIEW

File: src/views/Dashboard/Customer/RequestTestCheckout/index.vue [hunk 1]
```diff
@@ -111,7 +111,7 @@
       :show="sendPaymentDialog"
       :lab="lab"
       :products="products"
-      :totalPrice="totalPrice"
```
Comment (kevinjanada) [inline src/views/Dashboard/Customer/RequestTestCheckout/index.vue:114]: knapa namanya diganti ya mas?

**Review by kevinjanada — COMMENTED** [review]
Ada komen ya mas luthfi

File: src/views/Dashboard/Customer/RequestTestCheckout/index.vue (hunk 1 above)
Comment (luthfiazhari) [reply src/views/Dashboard/Customer/RequestTestCheckout/index.vue:114]: tadi saya mau nambahin totalPrice sama qcPrice nya mas. jadi saya pake namanya buat nyimpen datanya. memang seharusnya ga usah di ganti mas ?

Comment (kevinjanada) [reply src/views/Dashboard/Customer/RequestTestCheckout/index.vue:114]: Oke mas dah bener, aku salah nangkep

**Pull request merged**